NAMESPACE

```diff
@@ -110,13 +110,17 @@ S3method(as_polars_series,integer64)
 S3method(as_polars_series,list)
 S3method(as_polars_series,logical)
 S3method(as_polars_series,polars_data_frame)
+S3method(as_polars_series,polars_expr)
```

Comment:
I'm quite skeptical of this addition; Expression should be evaluated by context, and I don't feel comfortable with it suddenly being converted into a Series.

Reply:
The problem is that without it `pl$DataFrame(x = pl$date_range(...))` fails with `Unsupported class for `as_polars_series()`: polars_expr, polars_object`

They have an `eager` argument: https://github.com/pola-rs/polars/blob/dae6cd9abd9c15667ab86e573f114c57aa43b023/py-polars/polars/functions/range/datetime_range.py#L223

but we chose to remove that to avoid having a parameter that changes the output format.

Reply:
Is there any reason why you cannot use `pl$select()` instead of `pl$DataFrame()` there?

Reply:
Are there any such use cases in existing documentation?

Adding another function something like `pl$date_range_series()` would be better, I think.

Reply:
In my opinion, `pl$DataFrame` is usually just strict version of `pl$select`.
Wouldn't it be enough to show that to the user?

Incidentally, in the current r-polars, `pl$DataFrame` works differently than Python Polars' DataFrame constructor because it uses `pl$select` internally; Python Polars' DataFrame constructor does not accept Expr input and has no recycling rule.

Reply:
> Perhaps this hint from Python Polars would reassure you if something like this was included in the error message?

Done by 00005bf063ec6c436cd18d5dcd018a4d1e5cf5f7

Reply:
I also added `as_polars_series.polars_expr` to ensure that such discussions are not repeated in the future.
https://github.com/eitsupi/neo-r-polars/blob/ba5e1ff3439865c8a8591ef4f8bca2e0c930f4cf/R/as_polars_series.R#L159-L169

Reply:
Ok the fact that `pl.DataFrame` also errors with Expr is good. 

> Adding another function something like pl$date_range_series() would be better, I think.

Not sure it's worth it if pl$select() + pl$date_range() work.